### PR TITLE
Edge. Final changes to 10.11 doc.

### PR DESCRIPTION
**Pull request by sharath-jk**

### CODE REVIEW

File: content/edge/migration.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 weight: 60
-title: Migrating Cumulocity IoT Edge
+title: Migrating Cumulocity IoT Edge 10.7 to 10.9
```
Comment (BeateRixen) [inline content/edge/migration.md:3]: Didn´t you want to remove this file from 10.11 and later versions (develop)?

File: content/edge/installation-bundle/installing-edge-using-gui.md [hunk 2]
```diff
@@ -4,10 +4,12 @@ title: Installing Edge using the GUI
...
 ---
 
-To install {{< product-c8y-iot >}} Edge using the GUI:
+To install {{< product-c8y-iot >}} Edge using the user interface:
 
 1. Connect and start the Edge appliance in the hypervisor. Wait until the network configuration screen appears.
-<img src="/images/edge/edge-network-configurator.png" name="Network Configurator" style="width:75%;"/>
+  <img src="/images/edge/edge-network-configurator.png" name="Network Configurator" style="width:75%;"/>
+
+  > **Note:** The network configuration blue screen does not appear post installation. After the installation, you must use the Administration application or the REST API to configure the network.
```
Comment (BeateRixen) [inline content/edge/installation-bundle/installing-edge-using-gui.md:12]: **Note:** > Info

File: content/edge/installation-bundle/installing-edge-using-gui.md [hunk 3]
```diff
@@ -4,10 +4,12 @@ title: Installing Edge using the GUI
 layout: redirect
 ---
 
-To install {{< product-c8y-iot >}} Edge using the GUI:
+To install {{< product-c8y-iot >}} Edge using the user interface:
```
Comment (BeateRixen) [inline content/edge/installation-bundle/installing-edge-using-gui.md:7]: The title still says GUI?

File: content/edge/migration.md (hunk 1 above)
Comment (sharath-jk) [reply content/edge/migration.md:3]: No. Pavan suggested we keep it to help customers in 10.7.

File: content/edge/installation-bundle/installing-edge-using-gui.md (hunk 2 above)
Comment (sharath-jk) [reply content/edge/installation-bundle/installing-edge-using-gui.md:12]: Done.

File: content/edge/installation-bundle/installing-edge-using-gui.md (hunk 3 above)
Comment (sharath-jk) [reply content/edge/installation-bundle/installing-edge-using-gui.md:7]: Done.

**Review by BeateRixen — APPROVED** [review]

**Review by MWindel — APPROVED** [review]

**Pull request merged**